services/online_exam_service/.env

```
@@ -1,18 +1,54 @@
 # SERVER CONFIG
 PORT=3000
+HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOST key should go before the PORT key

(UnorderedKey)